components/02-elements/popup/popup.js

```diff
@@ -0,0 +1,112 @@
+(function() {
+
+  'use strict';
+
+  const popupsButtons = [...document.querySelectorAll('.popup__button')];
+
+  popupsButtons.forEach(popupButton => {
+    class Popup {
```

Comment:
The class should be outside the loop and you should just create new instances inside.

Reply:
moved outside the loop

Reply:
tested on two same popups in header and working fine